ipc/provider/src/manager/evm/error_parsing.rs

```diff
@@ -0,0 +1,62 @@
+// Copyright 2022-2024 Protocol Labs
+// SPDX-License-Identifier: MIT
+
+use async_trait::async_trait;
+use ethers::providers::{Http, HttpClientError, JsonRpcClient};
+use ipc_actors_abis::error_parser::ContractErrorParser;
+use serde::de::DeserializeOwned;
+use serde::Serialize;
+use std::fmt::{Debug, Formatter};
+
+/// A fvm contract revert parsing util
+#[derive(Clone)]
+pub struct FvmHttp {
```

Comment:
Strictly speaking this is not by no means related to Fvm right? It is for EVM contracts.

Reply:
changed!